styles/nixos-manual/catppuccin.user.css

```diff
@@ -0,0 +1,201 @@
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    body {
+      background-color: @base !important;
+      color: @text;
+    }
+
+    p,
+    ol,
+    ul {
+      color: @text !important;
+    }
+
+    a {
+      border-bottom: 1px solid @accent-color !important;
+      color: @accent-color;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+      color: @accent-color;
+    }
+    h5 {
+      color: fade(@accent-color, 85);
+    }
+
+    div.variablelist {
+      color: @text;
+    }
+
+    code {
+      &.programlisting.hljs {
+        background-color: fade(@blue, 5);
+        border-radius: 0.5rem;
+        border: 1px solid transparent;
+        color: @lavender;
+
+        .hljs-attr {
+          color: @rosewater;
+        }
+
+        .hljs-keyword {
+          color: @mauve;
+        }
+
+        .hljs-string,
+        .hljs-char.escape_ {
+          color: @green;
+        }
+
+        .hljs-built_in {
+          color: @lavender;
+        }
+
+        .hljs-comment,
+        .hljs-doctag {
+          color: fade(@text, 40);
+        }
+
+        .hljs-meta.prompt_ {
+          color: fade(@blue, 50);
+        }
+
+        .hljs-number,
+        .hljs-literal {
+          color: @peach;
+        }
+
+        .hljs-subst,
+        .hljs-variable {
+          color: fade(@blue, 70);
+        }
+
+        .hljs-function .hljs-title {
+          color: @lavender;
+        }
```

Comment:
See https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-code-blocks--syntax-highlighting.